emscripten/pyconfig.h

```diff
@@ -0,0 +1,1740 @@
+/* pyconfig.h.  Generated from pyconfig.h.in by configure.  */
```

Comment:
I'm curious what this file is used for?

Reply:
It's only needed for `rust_with_cffi` which doesn't work because I didn't get the headers quite right anyways. I'm going to work on getting Pyodide out-of-tree builds working which should give us a standardized way to get the right headers.